layouts/default.html

```diff
@@ -19,7 +19,7 @@
     <div id="header-wrapper">
       <div id="header">
         <div>
-          <a class="logo" href="/"><img src="/images/logo.png" alt="SeeClickFix API" /></a>
+          <a class="logo" href="/"><img src="/images/seeclickfix-311-crm-logo.svg" alt="SeeClickFix API" height="33px"/></a>
           <ul class="nav">
```

Comment:
While we are here should we consider a "better" `alt` text?
Is there a current organization-wide `alt` text for logos?

Reply:
"SeeClickFix API" is definitely incorrect in this context. Even "SeeClickFix Home" or similar would be more accurate.